signing_service/signing_service/signing_service.py

```diff
@@ -124,6 +129,9 @@ def _validate_arguments(self):
         if self.initial_reconnect_delay < 0:
             raise SigningServiceValidationError('reconnect_delay must be non-negative integer.')
 
+        if not is_valid_public_key(self.concent_public_key):
+            raise SigningServiceValidationError('concent_public_key is not valid public key.')
```

Comment:
I'd call this exception differently because this is parameter validation and soon you'll also have validation for data that comes over the connection. These errors should be processed very differently - data validation should crash the application if it for any reason reaches the outer handler because it's never supposed to reach that point and it's a bug.